src/backup/BackupQuiz.tsx

```diff
@@ -73,15 +73,12 @@ const mapStateToProps = (state: RootState): StateProps => {
 }
 
 export const navOptionsForQuiz = ({ route }: OwnProps) => {
-  const navigatedFromSettings = route.params?.navigatedFromSettings
-  const onCancel = () => {
-    navigate(Screens.Settings)
-  }
+  const settingsScreen = route.params?.settingsScreen ?? undefined
   return {
     ...emptyHeader,
     headerLeft: () => {
-      return navigatedFromSettings ? (
-        <CancelButton onCancel={onCancel} style={styles.cancelButton} />
+      return settingsScreen ? (
+        <CancelButton onCancel={() => navigateHome()} />
```

Comment:
probably fine to go home here but the settingsScreen param now allows us to retain existing behavior right? We can navigate similar to what is done in BackupComplete?